feature/databricks-sql-warehouse-compatibility (#121)
fivetran/dbt_fivetran_log: CHANGELOG.md

# dbt_fivetran_log v1.7.1
[PR #121](https://github.com/fivetran/dbt_fivetran_log/pull/121) includes the following updates:

## Bug Fixes
- Users leveraging the Databricks SQL Warehouse runtime were previously unable to run the `fivetran_platform__audit_table` model due to an incompatible incremental strategy. As such, the following updates have been made:

>   - A new macro `is_databricks_sql_warehouse()` has been added to determine if a databricks runtime is a SQL Warehouse runtime for Databricks. This macro will return a boolean of `true` if the runtime is determined to be SQL Warehouse and `false` if it is any other runtime or destination.

Reviewer: ```suggestion
  - A new macro `is_databricks_sql_warehouse()` has been added to determine if a SQL Warehouse runtime for Databricks is being used. This macro will return a boolean of `true` if the runtime is determined to be SQL Warehouse and `false` if it is any other runtime or a non-Databricks destination.
```

>   - The above macro is used in determining the incremental strategy within the `fivetran_platform__audit_table`. For Databricks SQL Warehouses, there will be **no** incremental strategy used. All other destination runtime strategies are not impacted with this change.

Reviewer: ```suggestion
  - The above macro is used in determining the incremental strategy within the `fivetran_platform__audit_table`. For Databricks SQL Warehouses, there will be **no** incremental strategy used. All other destinations and runtime strategies are not impacted with this change.
```

>   - The above macro is used in determining the incremental strategy within the `fivetran_platform__audit_table`. For Databricks SQL Warehouses, there will be **no** incremental strategy used. All other destination runtime strategies are not impacted with this change.

Reviewer: should we also note that the macro is used to decide between delta and parquet file formats?

Author: Added the CHANGLEOG entry for this

>     - For the SQL Warehouse runtime, the best incremental strategy we could elect to use is the `merge` strategy. However, we do not have full confidence in the resulting data integrity of the output model when leveraging this strategy. Therefore, we opted for the model to replicate a full create or replace behavior for the time being.

Reviewer: ```suggestion
    - For the SQL Warehouse runtime, the best incremental strategy we could elect to use is the `merge` strategy. However, we do not have full confidence in the resulting data integrity of the output model when leveraging this strategy. Therefore, we opted for the model to be materialized as a non-incremental `table` for the time being.
```
